src/drawing.cpp

```diff
@@ -437,26 +405,11 @@ void ShowOnboardPixel()
 void IRAM_ATTR DrawLoopTaskEntry(void *)
 {
     debugW(">> DrawLoopTaskEntry\n");
-    
+
     // If this board has an onboard RGB pixel, set it up now
 
     PrepareOnboardPixel();
 
-    // Prep the matrix and its layers
-
-    #if USE_MATRIX
-        MatrixInit();
-        // We don't need color correction on the title layer, but we want it on the main background
-
-        LEDMatrixGFX::titleLayer.enableColorCorrection(false);
-        LEDMatrixGFX::backgroundLayer.enableColorCorrection(true);
-
-        // Starting the effect might need to draw, so we need to set the leds up before doing so
-        auto pMatrix = std::static_pointer_cast<LEDMatrixGFX>(g_ptrSystem->EffectManager().g());
-        pMatrix->setLeds(LEDMatrixGFX::GetMatrixBackBuffer());
-        auto spectrum = GetSpectrumAnalyzer(0);

```

Comment:
Most of this code is a duplicate of what was in MatrixInit(). As such, this is now also part of LEDMatrixGFX::InitializeHardware(). I removed the spectrum variable as it wasn't used anywhere.